packages/starlight/translations/pl.json

```diff
@@ -0,0 +1,30 @@
+{
+  "skipLink.label": "Przejdź do głównej zawartości",
+  "search.label": "Szukaj",
+  "search.shortcutLabel": "(Naciśnij /, aby wyszukać)",
+  "search.cancelLabel": "Anuluj",
+  "search.devWarning": "Wyszukiwanie jest dostępne tylko w buildach produkcyjnych. Spróbuj zbudować i uruchomić aplikację, aby przetestować lokalnie.",
+  "themeSelect.accessibleLabel": "Wybierz motyw",
+  "themeSelect.dark": "Ciemny",
+  "themeSelect.light": "Jasny",
+  "themeSelect.auto": "Auto",
+  "languageSelect.accessibleLabel": "Wybierz język",
+  "menuButton.accessibleLabel": "Menu",
+  "sidebarNav.accessibleLabel": "Główne",
+  "tableOfContents.onThisPage": "Na tej stronie",
+  "tableOfContents.overview": "Przegląd",
+  "i18n.untranslatedContent": "Ta treść nie jest jeszcze dostępna w Twoim języku.",
+  "page.editLink": "Edytuj stronę",
+  "page.lastUpdated": "Ostatnia aktualizacja:",
+  "page.previousLink": "Poprzednia strona",
+  "page.nextLink": "Następna strona",
+  "404.text": "Nie znaleziono. Sprawdź URL lub użyj wyszukiwarki.",
+  "aside.note": "Notatka",
+  "aside.tip": "Wskazówka",
+  "aside.caution": "Uwaga",
+  "aside.danger": "Ważne",
+  "fileTree.directory": "Folder",
```

Comment:
And again, the same question to make sure this is not something that was missed.

Reply:
Correct, as well. This was originally "Directory" ;)